.github/workflows/build.yml

```diff
@@ -63,7 +63,7 @@ jobs:
       matrix:
         goos: ["linux"]
         goarch: ["arm", "arm64", "386", "amd64"]
-        go: ["1.16"]
+        go: ["1.17.9"]
```

Comment:
Do we actually want to pin this to a patch version, or just 1.17 latest?
```suggestion
        go: ["1.17"]
```

Reply:
Going to just pin the patch for now, we can re-evaluate down the line if we so desire.